config/analytics.yml

```diff
@@ -1,11 +1,12 @@
 #
 # To integrate your app with Google Analytics, uncomment the lines below and add your API key information.
 #
-analytics:
```

Comment:
this template should still exists, we can just unset the variables it if we do not want tenants to fall back to a default. I'm not even sure we do want that?

Reply:
you mean we don't want the fallback? I agree, and had deleted this file actually. but it fails at [self.load_from_yaml](https://github.com/scientist-softserv/palni-palci/blob/54a7b255a1d71ac55da0e2ba624d0d264fdcf2b0/app/services/hyrax/analytics/google.rb#L33-L34) since the file doesn't exist. I saw that we have [analytics.yml commented out](https://github.com/samvera/hyku/blob/main/config/analytics.yml) in base hyku, and do not have a "google.rb" file/decorator, so I commented the file out too instead of trying to get around the failure.

would you prefer I try to get around the error?